src/main/java/org/apache/commons/imaging/formats/webp/chunks/WebPChunkVp8x.java

```diff
@@ -42,76 +44,105 @@
  * }</pre>
  *
  * @see <a href="https://developers.google.com/speed/webp/docs/riff_container#extended_file_format">Extended File Format</a>
- *
  * @since 1.0-alpha4
  */
-public final class WebPChunkVP8X extends WebPChunk {
-    private final boolean hasICC;
-    private final boolean hasAlpha;
-    private final boolean hasExif;
-    private final boolean hasXmp;
-    private final boolean isAnimation;
+public final class WebPChunkVp8x extends WebPChunk {
+    private final boolean icc;
```

Comment:
Should these field names be renamed? Referring to `PamFileInfo`, perhaps it would be better for us to keep the 'has' prefix in the field name.

Reply:
No strong opinion here, let me revert it.